uber/templates/regform.html

```diff
@@ -575,7 +575,7 @@ <h4>{{ tier.name }}</h4>
     <label for="promo_code" class="col-sm-3 control-label optional-field">Promo Code</label>
     <div class="col-sm-6">
         {% if attendee.is_unpaid %}
-        <input type="textarea" name="promo_code" value="{{ attendee.promo_code_code }}" class="form-control" placeholder="Promo Code">
+        <input type="textarea" name="promo_code" value="{{ attendee.promo_code_code or promo_code }}" class="form-control" placeholder="Promo Code">
```

Comment:
I wanted the PII error message to be the first one encountered. That means it has to happen before we attempt to add the promo code to the attendee. Which means the attendee won't have the promo code associated when the page renders. Thus, we must return `promo_code` in a variable to the page.

This has the added benefit of being able to distribute promo codes in a URL like: https://localhost:4443/uber/preregistration/form?promo_code=1234

Reply:
Oof, that is quite a complication. Really nice side effect to fixing it, though!

Reply:
Silly me! I realized after I posted that comment that promo code URLs actually worked just fine before this update! Still needed to make that change tho.

https://localhost:4443/uber/preregistration/form?promo_code=1234